hack/tools/Makefile

```diff
@@ -19,6 +19,8 @@ $(LOCALBIN)/ginkgo:
 $(LOCALBIN)/golangci-lint:
 	curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(LOCALBIN) v1.21.0
 
+$(LOCALBIN)/pub:
+	GOBIN=$(LOCALBIN) $(GO) get github.com/devigned/pub/...
```

Comment:
Might want to lock this down to a specific version like
```
GOBIN=$(LOCALBIN) $(GO) get github.com/devigned/pub/...@v0.2.0
```

Reply:
Sure - i wasn't sure if i needed to add that here as well as in go.mod.

Reply:
updated